src/NUnitFramework/framework/Assert.cs

```diff
@@ -343,7 +358,9 @@ private static string GetStackTrace()
         {
             string stackTrace = null;
 
-#if PORTABLE
+#if PORTABLE && !NETSTANDARD1_6
+            // TODO: This isn't actually working! Since we catch it right here,
```

Comment:
I didn't add an issue for this one yet. In fact, I can't figure out how to get a full stack trace in portable.

Do we want to fix PORTABLE or will we be getting away from it soon enough. Maybe lets get netstandard working and then decide if portable does anything for us.